test/e2e/metering_manual_install_tests.go

```diff
@@ -107,6 +108,149 @@ func testManualMeteringInstall(
 	assert.NoError(t, err, "capturing logs and uninstalling metering should produce no error")
 }
 
+func createNFSProvisioner(ctx *deployframework.DeployerCtx) error {
+
+	ctx.TargetPodsCount = nonHDFSTargetPodCount
```

Comment:
This needs to be updated to reflect the NFS provisioner Pod that will be deployed in the ctx.Namespace as `nonHDFSTargetPodCount` is set to an integer value of five, but we're going to be provisioning six pods in total in this testing namespace, so we risk marking the metering installation as "ready to run post-install tests" but something like the reporting-operator Pod won't be healthy yet, which would be problematic.